lib/std/child_process.zig

```diff
@@ -46,6 +46,12 @@ pub const ChildProcess = struct {
 
     /// Set to change the current working directory when spawning the child process.
     cwd: ?[]const u8,
+    /// Set to change the current working directory when spawning the child process.
+    /// This is not yet implemented for Windows. See https://github.com/ziglang/zig/issues/5190
+    /// Once that is done, `cwd` will be deprecated in favor of this field.
+    /// The directory handle must be opened with the ability to be passed
+    /// to a child process (no `O_CLOEXEC` flag on POSIX).
```

Comment:
This will cause races: you should always remove the `O_CLOEXEC` in the forked process (before the exec)

Reply:
Can you explain or link to reading material?

Reply:
Take a situation where 2 threads simultaneously spawn a child process: you don't want one messing with fds to affect the other ==> you need to do all the fd manipulation in the child (after fork; before exec). `posix_spawn` was invented to solve this problem (as its very hard to do correctly with respect to signals).

Reply:
Ahh that makes sense, thanks! Would you mind making this note on #5190 ?